### ISSUE TITLE
Module not found: Error: Can't resolve '@emotion/styled/base'

### ISSUE THREAD

**Issue by ruslan-byondxr** [nrwl/nx#6814]
## Current Behavior

Storybook throwing error after generating storybook for react lib with emotion ,.

```
Module not found: Error: Can't resolve '@emotion/styled/base'
```

## Expected Behavior

Should run as expected

## Steps to Reproduce

Generate react lib with @emotion:

```
nx generate @nrwl/react:library --name=some-lib --style=@emotion/styled --linter=eslint --unitTestRunner=jest --component --strict --no-interactive
```

Generate storybook react configuration:

```
nx generate @nrwl/react:storybook-configuration --name=some-lib --configureCypress --generateStories --generateCypressSpecs --linter=eslint --no-interactive
```

Run generated storybook:

```
 nx run some-lib:storybook
 ```

### Failure Logs

``` 
🔴  ERROR in ./libs/some-lib/src/lib/some-lib.tsx
🔴  Module not found: Error: Can't resolve '@emotion/styled/base' in '/Users/user/Desktop/byondxr-projects/byondxr-monorepo/libs/some-lib/src/lib'
```

### Environment

Node : 14.17.4
  OS   : darwin x64
  yarn : 1.22.11
  
  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 12.7.2
  @nrwl/cypress : 12.7.2
  @nrwl/devkit : 12.7.2
  @nrwl/eslint-plugin-nx : 12.7.2
  @nrwl/express : Not Found
  @nrwl/jest : 12.7.2
  @nrwl/linter : 12.7.2
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 12.7.2
  @nrwl/nx-cloud : Not Found
  @nrwl/react : 12.7.2
  @nrwl/schematics : Not Found
  @nrwl/tao : 12.7.2
  @nrwl/web : 12.7.2
  @nrwl/workspace : 12.7.2
  @nrwl/storybook : 12.7.2
  @nrwl/gatsby : Not Found
  typescript : 4.3.5

**Comment by ruslan-byondxr** [comment]
works fine If using  styled-components instead

**Comment by puku0x** [comment]
Fortunately, https://github.com/nrwl/nx/pull/6787 has been merged and this issue will be solved soon in Nx v12.8.0.

**Comment by evanjmg** [comment]
Still an issue for me in 2023 on 14.3.6
